server/common/transport-shared/src/main/java/org/kaaproject/kaa/server/transport/channel/ChannelAware.java

```diff
@@ -13,31 +13,36 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package org.kaaproject.kaa.server.transport.channel;
 
-package org.kaaproject.kaa.server.common.server;
+import java.util.UUID;
 
 /**
- * Interface Track is used to collect statistics of requests.
- *
- * @author Andrey Panasenko
+ * Marker interface indicating that a class has a basic knowledge about related
```

Comment:
A marker interface indicating that the class has a basic knowledge about the related channel - it's {@link UUID}, {@link ChannelType} and {@link ChannelContext}.